### ISSUE TITLE
attr-name-style rule default is "lowercase"

### ISSUE THREAD

**Issue by WenryXu** [htmllint/htmllint#217]
But in Wiki https://github.com/htmllint/htmllint/wiki/Options#attr-name-style , default is "dash"

**Comment by WenryXu** [comment]
.htmllintrc
```
{
}
```

index.html
```html
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Document</title>
</head>
<body>

</body>
</html>
```

htmllint index.html
```
index.html: line 5, col 6, attribute names must match the format: lowercase
```

**Comment by mlochbaum** [comment]
The default has been changed in the latest version, 0.7.0. The wiki only ever has documentation for the current Github master branch, unfortunately. Maybe I should look into shipping documentation with the package?

Incidentally, the options page on the wiki is generated, so it can't really be wrong provided it's kept up to date.

Closing this because the documentation is correct but comments are still welcome.
